### PR TITLE
add commitlint instructions and travis-ci config

### PR DESCRIPTION
**Pull request by pbredenberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,26 @@ repos.
...
-TODO: fill in details and examples here.
+### Commitlint
 
+* Add a file called commitlint.config.js to your project root with the following content:
+
+```javascript
+'use strict';
+
+module.exports = {
+   extends: [ '@silvermine/eslint-config/commitlint.js' ],
```
Comment (yokuze) [inline README.md:47]: We will move this commitlint config from eslint-config-silvermine to this repo.

Can you copy that file from there to here and update this PR? This instructions will need to be updated too.

File: README.md [hunk 2]
```diff
@@ -36,8 +36,26 @@ repos.
...
+```
+
+* Use git log --oneline to find the short hash of the previous commit and take note of it
+* Add the following NPM script to `package.json`:
+
+   `"commitlint": "commitlint --from deadbeef"` (where deadbeef is the short hash from the previous step)
+
+* Configure `.gitlab-ci.yml` or `.travis.yml` to run `npm run commitlint` before running `npm test`
+
+TODO: fill in more details and examples here
```
Comment (yokuze) [inline README.md:58]: We can probably remove this line now too.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  - "lts/*" # Latest LTS version
+  - "12"
+  - "10"
+  - "8"
+
+before_install: npm i -g npm@6.13.4
+
+script:
+   - commitlint-travis
+   - npm run test
```
Comment (yokuze) [inline .travis.yml:13]: No newline at the end of this file.

File: README.md (hunk 1 above)
Comment (yokuze) [reply README.md:47]: Eventually we will need a corresponding PR to eslint-config-silvermine that removes the config from there and sets it up to use the config from this repo.

File: README.md [hunk 4]
```diff
@@ -36,8 +36,26 @@ repos.
 
 ## Usage
 
-TODO: fill in details and examples here.
+### Commitlint
 
+* Add a file called commitlint.config.js to your project root with the following content:
```
Comment (yokuze) [inline README.md:41]: Could you run markdownlint with our standards against this file? I think these bullet points should be indented, and that would help catch any other errors.

Comment (pbredenberg) [reply README.md:41]: I was waiting to lint the markdown file until the markdown lint config is merged. 🐔 or 🥚

File: README.md (hunk 1 above)
Comment (pbredenberg) [reply README.md]: Jeremy initialized the repo with that file already inside. Docs are updated!

File: README.md (hunk 2 above)
Comment (pbredenberg) [reply README.md:58]: ADIRL we'll leave this for now since technically there's still some docs TODO :D

**Comment by pbredenberg** [comment]
@yokuze updated per your comments

**Pull request merged**